### DISCUSSION TITLE
Webpage wallpaper with video stops system sleep/screensaver

### DISCUSSION THREAD

**Discussion by DianaSamayoa** [rocksdanister/lively#1586] [Q&A] [answered]
Hi! I made a custom wallpaper HTML with a video and Forecast/clock if that is relevant. 
But My OS (Windows 10)  never shuts down the monitor or starts the screen saver mode (even the one downloaded from here) when the app is running in the first plane or with a window open. I've read the app can only use the same wallpaper as a screen saver but it doesn't even do that. 
 Tried:
 - Changing the configuration of the screen saver and waiting time from windows
 - Using the SCR. file from here
 - Repairing windows with CMD using  Run SFC Scan and Run DISM - but closing the app allows the screen saver to work only with lively Wallpaper open it doesn't work.

**Comment by rocksdanister** [comment, marked as answer]
If the video on the html as audio track the audio drivers may block sleep so try removing the audio track.

You can check what's stopping system sleep by running `powercfg -requests` in terminal.

**Reply by DianaSamayoa** [reply]
![image](https://user-images.githubusercontent.com/70174776/220985054-d12c6877-1c31-4cd3-9407-aefc73b3f948.png)

Hi! thanks for your answer...here's what I got. But the video perse doesn't have an audio track. Maybe its how I'm calling the video?

**Reply by DianaSamayoa** [reply]
Hi!! I solved it Apparently this issue also happens in Wallpaper Engine too lol. 
I guess the logic is the same as the fullScreen videos on Youtube prevent the computer to sleep 
https://help.wallpaperengine.io/en/general/brokensleep.html#change-the-windows-power-settings
